core/utils.py

```diff
@@ -1179,3 +1182,18 @@ def quoted(s: str) -> str:
         str. The quoted string.
     """
     return json.dumps(s)
+
+
+def url_open(source_url: str) -> Any:
+    """Opens a URL and returns the response.
+
+    Args:
+        source_url: str. The URL.
+
+    Returns:
+        urlopen. The 'urlopen' object.
+    """
+    # TODO(#12912): Remove pylint disable after the arg-name-for-non-keyword-arg
+    # check is refactored.
+    context = ssl.create_default_context(cafile=certifi.where())  # pylint: disable=arg-name-for-non-keyword-arg
+    return urlrequest.urlopen(source_url, context=context)
```

Comment:
```suggestion
    # The type ignore is needed, because typestubs define the return type
    # of 'urlopen' as 'Any' which is wrong.
    return urllib.request.urlopen(source_url, context=context) # type: ignore[no-any-return] 
```

Reply:
it show `unused 'type: ignore' comment`, should i remove type: ignore and code comment above return statement?

Reply:
Ah, I forgot that the return type of this function should be str in order to get this working correctly.

Reply:
Also, not that the tests are failing.

Reply:
Done, remove code comment.

Reply:
Why remove the comment, change the return type of this function to `str` and add the ignore + comment.

Also, tests are still failing.

Reply:
Done, my fault.